ocaml/libs/cpuid/dune

```diff
@@ -0,0 +1,8 @@
+(library
+ (name cpuid)
+ (public_name cpuid)
```

Comment:
If you only use it inside xenopsd you don't have to make it a public library (simplifies the spec file as you won't need to install it). Then you also won't need an opam file for it.

Reply:
I needed it public for the unit tests at one point, but perhaps not anymore... Let me check...

Reply:
Yep, I can make it non-public.